eden/scm/lib/backingstore/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@
 # This software may be used and distributed according to the terms of the
 # GNU General Public License version 2.
 
-rust_static_library(rust_backingstore CRATE backingstore FEATURES fb)
```

Comment:
Can you make this optional? I think we have a CMake variable to check if it is Facebook internal build or not.